swift/Samples/TicTacToe/Sources/Game/ConfirmQuitWorkflow.swift

```diff
@@ -0,0 +1,95 @@
+/*
+* Copyright 2019 Square Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import Workflow
+import WorkflowUI
+import ReactiveSwift
+import BackStackContainer
+
+
+// MARK: Input and Output
+
+struct ConfirmQuitWorkflow: Workflow {
+    
+   // let baseScreen: AnyScreen
+    
+   enum Output {
+        case cancel
+        case confirm
+    }
+
+}
+
+
+// MARK: State and Initialization
+
+extension ConfirmQuitWorkflow {
+
+    typealias State = Void
+
+    func makeInitialState() -> ConfirmQuitWorkflow.State {
+        return State()
+    }
```

Comment:
@bencochran opened a PR recently that handles this for you if `State = Void`. (But maybe it hasn't been merged yet)

Reply:
I was not aware of that. I can keep an eye out for it. Thanks for letting me know.

Reply:
It landed here: https://github.com/square/workflow/pull/1094